src/Components/PaymentInputField.res

```diff
@@ -105,7 +109,9 @@ let make = (
           name
           ?maxLength
           ?pattern
-          className={`Input ${inputClass} ${className} focus:outline-none transition-shadow ease-out duration-200`}
+          className={`${innerLayout === Spaced
```

Comment:
Please move this condition outside as well

Reply:
Updated